ci-viewer.members.txt

```diff
@@ -5,3 +5,4 @@ hejinchi@cn.ibm.com
 holden.karau@gmail.com
 iman.tabrizian@gmail.com
 redhat-team@kubeflow.org
+nrchakradhar379@gmail.com
```

Comment:
alphabetical order please

Reply:
Done. Sorry. I thought it was chronological.